### PR TITLE
[TS migration] Migrate 'CustomStatus' page to TypeScript

### PR DESCRIPTION
**Pull request by war-in**
### Fixed Issues

$ https://github.com/Expensify/App/issues/38540
PROPOSAL:

### Tests

- [x] Go to profile page (`/settings/profile`)
- [x] Click `Status` section (`/settings/profile/status`)
- [x] Set some status
- [x] Clear the status

### QA Steps

- [ ] Go to profile page (`/settings/profile`)
- [ ] Click `Status` section (`/settings/profile/status`)
- [ ] Set some status
- [ ] Clear the status

### PR Author Checklist

- [x] I linked the correct issue in the `### Fixed Issues` section above
- [x] I wrote clear testing steps that cover the changes made in this PR
    - [x] I added steps for local testing in the `Tests` section
    - [x] I added steps for the expected offline behavior in the `Offline steps` section
    - [x] I added steps for Staging and/or Production testing in the `QA steps` section
    - [x] I added steps to cover failure scenarios (i.e. verify an input displays the correct error message if the entered data is not correct)
    - [x] I turned off my network connection and tested it while offline to ensure it matches the expected behavior (i.e. verify the default avatar icon is displayed if app is offline)
    - [x] I tested this PR with a [High Traffic account](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#high-traffic-accounts) against the staging or production API to ensure there are no regressions (e.g. long loading states that impact usability).
- [x] I included screenshots or videos for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [x] I ran the tests on **all platforms** & verified they passed on:
    - [x] Android: Native
    - [x] Android: mWeb Chrome
    - [x] iOS: Native
    - [x] iOS: mWeb Safari
    - [x] MacOS: Chrome / Safari
    - [x] MacOS: Desktop
- [x] I verified there are no console errors (if there's a console error not related to the PR, report it or open an issue for it to be fixed)
- [x] I followed proper code patterns (see [Reviewing the code](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [x] I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. `toggleReport` and not `onIconClick`)
    - [x] I verified that the left part of a conditional rendering a React component is a boolean and NOT a string, e.g. `myBool && <MyComponent />`.
    - [x] I verified that comments were added to code that is not self explanatory
    - [x] I verified that any new or modified comments were clear, correct English, and explained "why" the code was doing something instead of only explaining "what" the code was doing.
    - [x] I verified any copy / text shown in the product is localized by adding it to `src/languages/*` files and using the [translation method](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60)
      - [x] If any non-english text was added/modified, I verified the translation was requested/reviewed in #expensify-open-source and it was approved by an internal Expensify engineer. Link to Slack message:
    - [x] I verified all numbers, amounts, dates and phone numbers shown in the product are using the [localization methods](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60-L68)
    - [x] I verified any copy / text that was added to the app is grammatically correct in English. It adheres to proper capitalization guidelines (note: only the first word of header/labels should be capitalized), and is either coming verbatim from figma or has been approved by marketing (in order to get marketing approval, ask the Bug Zero team member to add the Waiting for copy label to the issue)
    - [x] I verified proper file naming conventions were followed for any new files or renamed files. All non-platform specific files are named after what they export and are not named "index.js". All platform-specific files are named for the platform the code supports as outlined in the README.
    - [x] I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#jsdocs)) were followed
- [x] If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers
- [x] I followed the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md)
- [x] I tested other components that can be impacted by my changes (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` are working as expected)
- [x] I verified all code is DRY (the PR doesn't include any logic written more than once, with the exception of tests)
- [x] I verified any variables that can be defined as constants (ie. in CONST.js or at the top of the file that uses the constant) are defined as such
- [x] I verified that if a function's arguments changed that all usages have also been updated correctly
- [x] If any new file was added I verified that:
    - [x] The file has a description of what it does and/or why is needed at the top of the file if the code is not self explanatory
- [x] If a new CSS style is added I verified that:
    - [x] A similar style doesn't already exist
    - [x] The style can't be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/styles/utils/index.ts) function (i.e. `StyleUtils.getBackgroundAndBorderStyle(theme.componentBG)`)
- [x] If the PR modifies code that runs when editing or sending messages, I tested and verified there is no unexpected behavior for all supported markdown - URLs, single line code, code blocks, quotes, headings, bold, strikethrough, and italic.
- [x] If the PR modifies a generic component, I tested and verified that those changes do not break usages of that component in the rest of the App (i.e. if a shared library or component like `Avatar` is modified, I verified that `Avatar` is working as expected in all cases)
- [x] If the PR modifies a component related to any of the existing Storybook stories, I tested and verified all stories for that component are still working as expected.
- [x] If the PR modifies a component or page that can be accessed by a direct deeplink, I verified that the code functions as expected when the deeplink is used - from a logged in and logged out account.
- [x] If the PR modifies the UI (e.g. new buttons, new UI components, changing the padding/spacing/sizing, moving components, etc) or modifies the form input styles:
    - [x] I verified that all the inputs inside a form are aligned with each other.
    - [x] I added `Design` label and/or tagged `@Expensify/design` so the design team can review the changes.
- [x] If a new page is added, I verified it's using the `ScrollView` component to make it scrollable when more elements are added to the page.
- [x] If the `main` branch was merged into this PR after a review, I tested again and verified the outcome was still expected according to the `Test` steps.

### Screenshots/Videos
<details>
<summary>Android: Native</summary>

https://github.com/Expensify/App/assets/61014013/fe2046e6-29f8-4ff7-a1c3-58d200844259

</details>

<details>
<summary>Android: mWeb Chrome</summary>

https://github.com/Expensify/App/assets/61014013/e908ed04-9ecf-4b60-b32a-74351c708044

</details>

<details>
<summary>iOS: Native</summary>

https://github.com/Expensify/App/assets/61014013/14d23f35-fce2-4a49-8b8c-1e6039049511

</details>

<details>
<summary>iOS: mWeb Safari</summary>

https://github.com/Expensify/App/assets/61014013/a6674306-9dfb-45bb-ac8f-cafabacaec02

</details>

<details>
<summary>MacOS: Chrome / Safari</summary>

https://github.com/Expensify/App/assets/61014013/74c2fe9c-351b-410e-bcf5-1b1ed994b78f

</details>

<details>
<summary>MacOS: Desktop</summary>

https://github.com/Expensify/App/assets/61014013/ace9fd6b-6fd1-45a6-bff4-10767a49bb37

</details>

### CODE REVIEW

File: src/pages/settings/Profile/CustomStatus/StatusClearAfterPage.tsx [hunk 1]
```diff
@@ -156,7 +148,7 @@ function StatusClearAfterPage({currentUserPersonalDetails, customStatus}) {
 
     const timePeriodOptions = useCallback(
         () =>
-            _.map(statusType, (item) => (
+            Object.entries(statusType).map(([, item]) => (
```
Comment (blazejkustra) [inline src/pages/settings/Profile/CustomStatus/StatusClearAfterPage.tsx:151]: `Object.values` is a better fit here

File: src/pages/settings/Profile/CustomStatus/StatusClearAfterPage.tsx [hunk 2]
```diff
@@ -134,8 +126,8 @@ function StatusClearAfterPage({currentUserPersonalDetails, customStatus}) {
             if (mode.value === CONST.CUSTOM_STATUS_TYPES.CUSTOM) {
                 User.updateDraftCustomStatus({clearAfter: DateUtils.getOneHourFromNow()});
             } else {
-                const selectedRange = _.find(statusType, (item) => item.value === mode.value);
-                const calculatedDraftDate = DateUtils.getDateFromStatusType(selectedRange.value);
+                const selectedRange = statusType.find((item) => item.value === mode.value);
+                const calculatedDraftDate = DateUtils.getDateFromStatusType(selectedRange?.value ?? 'never');
```
Comment (blazejkustra) [inline src/pages/settings/Profile/CustomStatus/StatusClearAfterPage.tsx:130]: Why `'never'`?

File: src/pages/settings/Profile/CustomStatus/StatusClearAfterPage.tsx [hunk 3]
```diff
@@ -102,30 +94,30 @@ function StatusClearAfterPage({currentUserPersonalDetails, customStatus}) {
...
+            const selectedRange = statusType.find((item) => item.isSelected);
+            calculatedDraftDate = DateUtils.getDateFromStatusType(selectedRange?.value ?? 'never');
         }
         User.updateDraftCustomStatus({clearAfter: calculatedDraftDate});
         Navigation.goBack(ROUTES.SETTINGS_STATUS);
     };
 
     const updateMode = useCallback(
-        (mode) => {
+        (mode: {value: CustomStatusTypes}) => {
```
Comment (blazejkustra) [inline src/pages/settings/Profile/CustomStatus/StatusClearAfterPage.tsx:120]: Let's extract it to a separate type

File: src/pages/settings/Profile/CustomStatus/StatusClearAfterPage.tsx [hunk 4]
```diff
@@ -102,30 +94,30 @@ function StatusClearAfterPage({currentUserPersonalDetails, customStatus}) {
...
         }
-        let calculatedDraftDate = '';
+        let calculatedDraftDate;
         if (draftPeriod === CONST.CUSTOM_STATUS_TYPES.CUSTOM) {
             calculatedDraftDate = draftClearAfter;
         } else {
-            const selectedRange = _.find(statusType, (item) => item.isSelected);
-            calculatedDraftDate = DateUtils.getDateFromStatusType(selectedRange.value);
+            const selectedRange = statusType.find((item) => item.isSelected);
+            calculatedDraftDate = DateUtils.getDateFromStatusType(selectedRange?.value ?? 'never');
```
Comment (blazejkustra) [inline src/pages/settings/Profile/CustomStatus/StatusClearAfterPage.tsx:113]: Same here, why `'never'`?

File: src/pages/settings/Profile/CustomStatus/StatusClearAfterPage.tsx [hunk 5]
```diff
@@ -102,30 +94,30 @@ function StatusClearAfterPage({currentUserPersonalDetails, customStatus}) {
...
-            redBrickTimeIndicator: customTimeError ? CONST.BRICK_ROAD_INDICATOR_STATUS.ERROR : null,
+            redBrickDateIndicator: customDateError ? CONST.BRICK_ROAD_INDICATOR_STATUS.ERROR : undefined,
+            redBrickTimeIndicator: customTimeError ? CONST.BRICK_ROAD_INDICATOR_STATUS.ERROR : undefined,
         }),
         [customTimeError, customDateError],
     );
 
     const onSubmit = () => {
-        const {dateError, timeError} = validateCustomDate();
+        const {dateValidationErrorKey: dateError, timeValidationErrorKey: timeError} = validateCustomDate();
```
Comment (blazejkustra) [inline src/pages/settings/Profile/CustomStatus/StatusClearAfterPage.tsx:104]: This is a big change in logic, this wasn't working before?

File: src/pages/settings/Profile/CustomStatus/StatusClearAfterPage.tsx [hunk 6]
```diff
@@ -102,30 +94,30 @@ function StatusClearAfterPage({currentUserPersonalDetails, customStatus}) {
...
         if (dateError || timeError) {
             return;
         }
-        let calculatedDraftDate = '';
+        let calculatedDraftDate;
```
Comment (blazejkustra) [inline src/pages/settings/Profile/CustomStatus/StatusClearAfterPage.tsx:108]: Let's add a type here, as of now it is typed as any

Comment (blazejkustra) [reply src/pages/settings/Profile/CustomStatus/StatusClearAfterPage.tsx:108]: Or leave an empty string, to avoid regressions

File: src/pages/settings/Profile/CustomStatus/StatusClearAfterPage.tsx [hunk 7]
```diff
@@ -1,59 +1,49 @@
...
  */
-function getSelectedStatusType(data) {
+function getSelectedStatusType(data: string): CustomStatusTypes {
     switch (data) {
         case DateUtils.getEndOfToday():
             return CONST.CUSTOM_STATUS_TYPES.AFTER_TODAY;
         case CONST.CUSTOM_STATUS_TYPES.NEVER:
         case '':
             return CONST.CUSTOM_STATUS_TYPES.NEVER;
-        case false:
```
Comment (blazejkustra) [inline src/pages/settings/Profile/CustomStatus/StatusClearAfterPage.tsx:49]: Are you sure this isn't used?

File: src/pages/settings/Profile/CustomStatus/StatusClearAfterPage.tsx [hunk 8]
```diff
@@ -1,59 +1,49 @@
...
-const propTypes = {
-    currentUserPersonalDetails: personalDetailsPropType,
-    customStatus: PropTypes.shape({
-        clearAfter: PropTypes.string,
-    }),
+type StatusClearAfterPageOnyxProps = {
+    customStatus: OnyxEntry<OnyxTypes.CustomStatusDraft>;
 };
 
+type StatusClearAfterPageProps = StatusClearAfterPageOnyxProps;
```
Comment (blazejkustra) [inline src/pages/settings/Profile/CustomStatus/StatusClearAfterPage.tsx:29]: Good job!

File: src/pages/settings/Profile/CustomStatus/StatusClearAfterPage.tsx [hunk 9]
```diff
@@ -1,59 +1,49 @@
...
-import personalDetailsPropType from '@pages/personalDetailsPropType';
 import CONST from '@src/CONST';
 import ONYXKEYS from '@src/ONYXKEYS';
 import ROUTES from '@src/ROUTES';
+import type * as OnyxTypes from '@src/types/onyx';
 
-const defaultProps = {
-    ...withCurrentUserPersonalDetailsDefaultProps,
-};
+type CustomStatusTypes = (typeof CONST.CUSTOM_STATUS_TYPES)[keyof typeof CONST.CUSTOM_STATUS_TYPES];
```
Comment (blazejkustra) [inline src/pages/settings/Profile/CustomStatus/StatusClearAfterPage.tsx:23]: Let's use util from typ-fest (ValueOf)

File: src/pages/settings/Profile/CustomStatus/SetDatePage.tsx [hunk 10]
```diff
@@ -58,7 +61,6 @@ function SetDatePage({translate, customStatus}) {
                 submitButtonText={translate('common.save')}
                 validate={validate}
                 enabledWhenOffline
-                shouldUseDefaultValue
```
Comment (blazejkustra) [inline src/pages/settings/Profile/CustomStatus/SetDatePage.tsx:61]: Why was this removed?

File: src/pages/settings/Profile/CustomStatus/SetDatePage.tsx [hunk 11]
```diff
@@ -1,38 +1,41 @@
...
+type SetDatePageProps = SetDatePageOnyxProps;
+
+function SetDatePage({customStatus}: SetDatePageProps) {
     const styles = useThemeStyles();
-    const customClearAfter = lodashGet(customStatus, 'clearAfter', '');
+    const {translate} = useLocalize();
+    const customClearAfter = customStatus?.clearAfter ?? '';
 
-    const onSubmit = (v) => {
+    const onSubmit = (v: {dateTime: string}) => {
```
Comment (blazejkustra) [inline src/pages/settings/Profile/CustomStatus/SetDatePage.tsx:32]: Nit: Extract this  to a separate type and rename `v` to `value`?

**Review by blazejkustra — CHANGES_REQUESTED** [review]
Couple comments, overall good work!

File: src/pages/settings/Profile/CustomStatus/StatusClearAfterPage.tsx (hunk 7 above)
Comment (war-in) [reply src/pages/settings/Profile/CustomStatus/StatusClearAfterPage.tsx:49]: The only place where `getSelectedStatusType` is used is in this file in `useState` where two strings are passed, so I'm quite sure

File: src/pages/settings/Profile/CustomStatus/StatusClearAfterPage.tsx (hunk 2 above)
Comment (war-in) [reply src/pages/settings/Profile/CustomStatus/StatusClearAfterPage.tsx:130]: I thought that if there is no `selectedRange` we shouldn't clear the status

File: src/pages/settings/Profile/CustomStatus/StatusClearAfterPage.tsx (hunk 5 above)
Comment (war-in) [reply src/pages/settings/Profile/CustomStatus/StatusClearAfterPage.tsx:104]: Is it? I thought that I was just renaming the fields 😅

File: src/pages/settings/Profile/CustomStatus/StatusClearAfterPage.tsx (hunk 7 above)
Comment (blazejkustra) [reply src/pages/settings/Profile/CustomStatus/StatusClearAfterPage.tsx:49]: Let's test manually and see if `data` argument is ever a boolean

File: src/pages/settings/Profile/CustomStatus/SetDatePage.tsx [hunk 12]
```diff
@@ -1,38 +1,45 @@
...
 import Navigation from '@libs/Navigation/Navigation';
 import * as ValidationUtils from '@libs/ValidationUtils';
 import ONYXKEYS from '@src/ONYXKEYS';
 import ROUTES from '@src/ROUTES';
 import INPUT_IDS from '@src/types/form/SettingsStatusClearDateForm';
+import type * as OnyxTypes from '@src/types/onyx';
 
-const propTypes = {
-    ...withLocalizePropTypes,
+type Date = {
```
Comment (blazejkustra) [inline src/pages/settings/Profile/CustomStatus/SetDatePage.tsx:21]: `Date` may be conflicting with the `Date` class, maybe a different name would fit better here? Perhaps DateTime? 😄

File: src/pages/settings/Profile/CustomStatus/SetDatePage.tsx [hunk 13]
```diff
@@ -58,31 +65,24 @@ function SetDatePage({translate, customStatus}) {
...
                 enabledWhenOffline
-                shouldUseDefaultValue
             >
                 <InputWrapper
                     InputComponent={DatePicker}
                     inputID={INPUT_IDS.DATE_TIME}
                     label={translate('statusPage.date')}
                     defaultValue={DateUtils.extractDate(customClearAfter)}
                     minDate={new Date()}
+                    shouldUseDefaultValue
```
Comment (blazejkustra) [inline src/pages/settings/Profile/CustomStatus/SetDatePage.tsx:75]: Let's test that it doesnt break anything

File: src/pages/settings/Profile/CustomStatus/SetTimePage.tsx [hunk 14]
```diff
@@ -40,6 +43,7 @@ function SetTimePage({translate, customStatus}) {
             />
             <View style={styles.flex1}>
                 <TimePicker
+                    // @ts-expect-error TODO: remove this comment once TimePicker (https://github.com/Expensify/App/issues/36129) is migrated to TS
```
Comment (blazejkustra) [inline src/pages/settings/Profile/CustomStatus/SetTimePage.tsx:46]: [TimePicker](https://github.com/Expensify/App/pull/37422) was just merged, merge the newest main and remove this comment

File: src/pages/settings/Profile/CustomStatus/StatusClearAfterPage.tsx [hunk 15]
```diff
@@ -1,59 +1,57 @@
...
+    isSelected: boolean;
 };
 
-const propTypes = {
-    currentUserPersonalDetails: personalDetailsPropType,
-    customStatus: PropTypes.shape({
-        clearAfter: PropTypes.string,
-    }),
+type StatusClearAfterPageOnyxProps = {
+    customStatus: OnyxEntry<OnyxTypes.CustomStatusDraft>;
```
Comment (blazejkustra) [inline src/pages/settings/Profile/CustomStatus/StatusClearAfterPage.tsx:34]: Let's add a JSDoc explaining this value

File: src/pages/settings/Profile/CustomStatus/StatusClearAfterPage.tsx (hunk 2 above)
Comment (blazejkustra) [reply src/pages/settings/Profile/CustomStatus/StatusClearAfterPage.tsx:138]: Let's use the constant then! `CONST.CUSTOM_STATUS_TYPES.NEVER`

File: src/pages/settings/Profile/CustomStatus/StatusClearAfterPage.tsx (hunk 5 above)
Comment (blazejkustra) [reply src/pages/settings/Profile/CustomStatus/StatusClearAfterPage.tsx:112]: Renaming works the other way around:
```ts
const {dateError: dateValidationErrorKey, timeError: timeValidationErrorKey} = validateCustomDate();
```

Looking at the previous code (mostly `validateCustomDate`) I assume this was a mistake in the logic.

Comment (blazejkustra) [reply src/pages/settings/Profile/CustomStatus/StatusClearAfterPage.tsx:112]: I think you can fix the other way around:
```ts
    const validate = () => {
        const {dateValidationErrorKey, timeValidationErrorKey} = ValidationUtils.validateDateTimeIsAtLeastOneMinuteInFuture(data);

        setCustomDateError(dateValidationErrorKey);
        setCustomTimeError(timeValidationErrorKey);

        return {
            dateError: dateValidationErrorKey,
            timeValidationErrorKey: timeValidationErrorKey,
        };
    };
```

**Review by blazejkustra — CHANGES_REQUESTED** [review]

File: src/pages/settings/Profile/CustomStatus/SetDatePage.tsx (hunk 13 above)
Comment (war-in) [reply src/pages/settings/Profile/CustomStatus/SetDatePage.tsx:75]: Checked manually and it seems that everything is fine

File: src/pages/settings/Profile/CustomStatus/StatusClearAfterPage.tsx (hunk 7 above)
Comment (war-in) [reply src/pages/settings/Profile/CustomStatus/StatusClearAfterPage.tsx:49]: Checked manually and it seems that it is always a string

**Review by blazejkustra — APPROVED** [review]

File: src/pages/settings/Profile/CustomStatus/SetTimePage.tsx [hunk 16]
```diff
@@ -40,24 +43,18 @@ function SetTimePage({translate, customStatus}) {
             />
             <View style={styles.flex1}>
                 <TimePicker
-                    inputID="timePicker"
                     defaultValue={clearAfter}
-                    style={styles.flexGrow1}
```
Comment (fabioh8010) [inline src/pages/settings/Profile/CustomStatus/SetTimePage.tsx:45]: Why was this props removed?

File: src/pages/settings/Profile/CustomStatus/StatusClearAfterPage.tsx [hunk 17]
```diff
@@ -1,59 +1,58 @@
...
+    /** User's custom status */
+    customStatus: OnyxEntry<OnyxTypes.CustomStatusDraft>;
 };
 
+type StatusClearAfterPageProps = StatusClearAfterPageOnyxProps;
+
 /**
- * @param {string} data -  either a value from CONST.CUSTOM_STATUS_TYPES or a dateTime string in the format YYYY-MM-DD HH:mm
- * @returns {string}
+ * @param data -  either a value from CONST.CUSTOM_STATUS_TYPES or a dateTime string in the format YYYY-MM-DD HH:mm
```
Comment (fabioh8010) [inline src/pages/settings/Profile/CustomStatus/StatusClearAfterPage.tsx:41]:
```suggestion
 * @param data - either a value from CONST.CUSTOM_STATUS_TYPES or a dateTime string in the format YYYY-MM-DD HH:mm
```

**Review by fabioh8010 — COMMENTED** [review]
Also, I have seen many changes unrelated to TS migration itself, let's test it extensively and make sure we won't have regressions!

File: src/pages/settings/Profile/CustomStatus/SetTimePage.tsx (hunk 16 above)
Comment (war-in) [reply src/pages/settings/Profile/CustomStatus/SetTimePage.tsx:45]: `TimePicker` does not accept `inputID` and `style` props after the migration

**Comment by war-in** [comment]
> Also, I have seen many changes unrelated to TS migration itself, let's test it extensively and make sure we won't have regressions!

@fabioh8010 could you point me to those changes? (or maybe you have in mind some tests you want to be performed)

**Review by fabioh8010 — APPROVED** [review]

**Comment by fabioh8010** [comment]
@war-in I think mostly these ones

https://github.com/Expensify/App/pull/38797/files#diff-a7880f31d36765762e93bccf6bbfb9a1198f37a5bbb34db7ac3cc57a9c111a8eR75
https://github.com/Expensify/App/pull/38797/files#diff-d1982517d848680c5ea7e4e39400f7a5a95e45f5f79649302865d2f85e784992L49
https://github.com/Expensify/App/pull/38797/files#diff-d1982517d848680c5ea7e4e39400f7a5a95e45f5f79649302865d2f85e784992R65-R66

but if you already tested them LGTM for me!

**Comment by war-in** [comment]
> @war-in I think mostly these ones
> 
> https://github.com/Expensify/App/pull/38797/files#diff-a7880f31d36765762e93bccf6bbfb9a1198f37a5bbb34db7ac3cc57a9c111a8eR75 https://github.com/Expensify/App/pull/38797/files#diff-d1982517d848680c5ea7e4e39400f7a5a95e45f5f79649302865d2f85e784992L49 https://github.com/Expensify/App/pull/38797/files#diff-d1982517d848680c5ea7e4e39400f7a5a95e45f5f79649302865d2f85e784992R65-R66
> 
> but if you already tested them LGTM for me!

@fabioh8010 yup, already tested 🫡

**Comment by fabioh8010** [comment]
@ntdiary kind bump 🙂

**Comment by war-in** [comment]
@ntdiary addressed!

**Review by ntdiary — APPROVED** [review]
LGTM. :)

**Review by ntdiary — APPROVED** [review]

**Review by arosiclair — APPROVED** [review]
